content/authors/Jianou Zhang/index.md

```diff
@@ -0,0 +1,41 @@
+---
+# Display name
+title: Jianou Zhang
+
+# Username (this should match your github username)
+authors:
+- Janeou-Zhang(Github name)
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role: graduate student/Fotech Solutions Project member
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Saskatchewan
+  url: http://www.usask.ca
+
+# A list of your interests, add as many as you want
+interests:
+- Bayesian Inferences
+- 
+
+# Your educational history, add as many as you want
+education: Master of Science in Statistics
```

Comment:
perhaps this info goes into line `28`? I think there should be nothing after `education:`